### ISSUE TITLE
[Info] How to declare variables across Tasks

### ISSUE THREAD

**Issue by robertofabrizi** [Netflix/conductor#551]
Dear friends,
I'm quite new to Conductor, I've created a simple Workflow that uses both the Worker Tasks and the Http Tasks, and it seems to be working just fine.

There is one thing though that I've not fully understood. Imagine just two Tasks in sequence, Task 1 and Task 2.
I've currently declared the output of the workflow as the result of Task 2, but what if Task 1 fails and therefore Task 2 is never called?

Thank you for your help!
Roberto

**Comment by cyzhao** [comment]
@robertofabrizi not sure I understand your question, it is expected that Task 2 is not called when Task 1 failed, what are you trying to achieve here?

**Comment by robertofabrizi** [comment]
@cyzhao  Sorry my bad I initially wrote a different sentence and forgot to re-add one bit. Imagine that in my list of N tasks they are declared non-blocking (iirc optional=true), therefore the following task does get called.

Lets see if I can make some proto-example in a programming language:

```
String output;
if(condition1 = true){
output="result 1";
}
if(condition2 = true){
output="result 2";
}
if(condition3 = true){
output="result 3";
}
return output;
```

Imagine that the ifs are the tasks, they all get executed / checked, and eventually override my final result that is returned at the end of the flow. How can I achieve this in conductor?

**Comment by cyzhao** [comment]
@robertofabrizi please take a look at fork join tasks, it may support what you need

**Comment by robertofabrizi** [comment]
@cyzhao That's more to achieve parallel execution of tasks rather than pass variables between serial tasks, isn't it?

**Comment by apanicker-nflx** [comment]
This seems similar to #953 which has been merged to dev in #979. We will release this soon once testing is completed.
